src/components/Table/Table.module.scss

```diff
@@ -15,6 +15,7 @@
   }
 
   .table {
+    margin-top: 1px;
```

Comment:
@JCarmine Why is this necessary?

Reply:
To make the focus ring rendered by Chrome fully visible; the ring's width is 2px, and it was partially obscured. We can chat about it in the am, and decide whether we care or not.